src/components/SwapInfo.vue

```diff
@@ -111,11 +111,15 @@ export default class SwapInfo extends Mixins(TranslationMixin) {
     return ''
   }
 
-  getAssetSymbolText (isXorSymbol = true): string {
-    if (isXorSymbol) {
-      return ' ' + KnownSymbols.XOR
+  get xorSymbol (): string {
+    return ' ' + KnownSymbols.XOR
+  }
+
+  get getAssetSymbolText (): string {
```

Comment:
How do you look at using the optional chaining and null coalescing operator, to shorten the checks?
```
get getAssetSymbolText (): string {
    if (this.isExchangeB) {
      return this.tokenFrom?.symbol ?? ''
    }
    return this.tokenTo?.symbol ?? ''
}
```

Reply:
I think it's better to have consistent approach in whole component, so I'll change it at refactoring stage later.